vue/.env

```diff
@@ -1 +1 @@
-API_BASE_URL=https://rapid-skills.herokuapp.com
\ No newline at end of file
+VUE_APP_API_BASE_URL=https://rapid-skills.herokuapp.com
```

Comment:
@jhash can we point this at the https://www.rapidskills.org/ url instead of the heroku app url?

Reply:
@Pcushing that's a question for @littleforest but it will also take whatever is set on the server itself as truth before this

Reply:
@jhash you can use https://www.rapidskills.org since it is set up with SSL now.